lib/tasks/portus.rake

```diff
@@ -89,4 +111,22 @@ HERE
     end
     puts
   end
+
+  desc "Properly test Portus"
+  task :test do |_, args|
+    tags = args.extras.map { |a| "--tag #{a}" }
+    tags << "--tag ~integration" if ENV["TRAVIS"] == "true"
```

Comment:
Not for now. I tried to do it (in multiple different ways) but I had lots of problems. I opened #847 for this same reason.